ingestion/functions/template.yaml

```diff
@@ -18,7 +18,7 @@ Resources:
     Properties:
       CodeUri: retrieval/
       Handler: retrieval.lambda_handler
-      Runtime: python3.6
+      Runtime: python3.8
```

Comment:
This is fine (can't recall exactly why 3.6 was used initially) -- but need to update the README.

Reply:
I updated the last references of it in the readme.